var/spack/repos/builtin/packages/libvips/package.py

```diff
@@ -32,6 +32,7 @@ class Libvips(AutotoolsPackage):
 
     # TODO: Add more variants!
 
+    depends_on("pkg-config", type="build")
```

Comment:
Do we really want/need `pkg-config` here or can we use the virtual `pkgconfig` which is more commonly used?

Reply:
I didn't actually try it with the virtual... that would probably be ok, but would need testing.

Reply:
Yes we should use the virtual

Reply:
```suggestion
    depends_on("pkgconfig", type="build")
```